sw/host/opentitanlib/src/test_utils/bootstrap.rs

```diff
@@ -16,15 +16,26 @@ pub struct Bootstrap {
     #[command(flatten)]
     pub options: BootstrapOptions,
 
-    /// RV32 test binary to load.
-    #[arg(long)]
-    pub bootstrap: Option<PathBuf>,
+    /// RV32 test binaries to load.
+    #[arg(long, num_args = 1..)]
+    pub bootstrap: Option<Vec<PathBuf>>,
 }
 
 impl Bootstrap {
     pub fn init(&self, transport: &TransportWrapper) -> Result<Option<Box<dyn Annotate>>> {
+        self.init_specific(transport, 0)?;
+        Ok(None)
+    }
+
+    pub fn init_specific(
```

Comment:
Why do you need to add a bootstrap function to be able to select from a list of items to bootstrap? We just want to bootstrap one image (as described above).

Reply:
It's because we only want to bootstrap perso fw when we do the first bootstrap to save time loading big chunks of firmware (combined image) into flash.

Reply:
Instead of creating a separate bootstrap function, can yo just add another command line arg like was done in the perso flow for the Z1 chip: https://github.com/lowRISC/opentitan/blob/3c72c0ed03608a03d1d0350c8e759a81ad55dfd7/sw/host/provisioning/ft/src/main.rs#L95 ?

Then you can just call the existing bootstrap function with this arg like so: https://github.com/lowRISC/opentitan/blob/3c72c0ed03608a03d1d0350c8e759a81ad55dfd7/sw/host/provisioning/ft_lib/src/lib.rs#L291

The earlgrey Z1 chip had 4 perso binaries due to size constraints set by the previous ROM. But those don't exist on the A1 chip so we just had one binary. But you can use the same approach we used before.

Reply:
okay, I follow the same approach to add another command arg to get the path of the assemble image.